### PR TITLE
Announcing Scala.js 0.6.4.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @nicolasstucki and all who want :-)

File: _posts/news/2015-07-03-announcing-scalajs-0.6.4.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Some Scala libraries use the collections of Java in some cases.
+To help port these libraries to Scala.js, a number of collection types have been ported.
+At the moment, the following data structures are supported:
+
+* `List`: `ArrayList`, `LinkedList`, `CopyOnWriteArrayList`
+* `Set`: `HashSet`, `LinkedHashSet`, `ConcurrentSkipListSet`
+* `Map`: `HashMap`, `LinkedHashMap`, `ConcurrentHashMap`
+* `Queue`: `LinkedList`, `ConcurrentLinkedQueue`
+
```
Comment (nicolasstucki) [inline _posts/news/2015-07-03-announcing-scalajs-0.6.4.md]: We should add a note on the simplifications in the `concurrent` collection due to single threading.

**Comment by nicolasstucki** [comment]
Thats all. Otherwise LGTM

File: _posts/news/2015-07-03-announcing-scalajs-0.6.4.md (hunk 1 above)
Comment (sjrd) [inline _posts/news/2015-07-03-announcing-scalajs-0.6.4.md]: Why? They completely respect their semantics. The simplifications are an implementation detail.

**Comment by gzm0** [comment]
You forgot about the changed `@JSExport` rules.

**Comment by sjrd** [comment]
Right. I'll add that.

**Comment by sjrd** [comment]
Updated.

**Comment by sjrd** [comment]
OK all ready. Will hit merge tomorrow morning CEST, before taking the train to the north of France.

**Pull request merged**